api/apps/api/src/modules/features/features/domain/feature.ts

```diff
@@ -0,0 +1,39 @@
+import { FeatureFragment } from './feature-fragment';
+import { CustomFeature } from '@marxan-api/modules/features/features/domain/custom-feature';
+
+export enum FeatureTag {
+  Species = 'species',
+  Bioregional = 'bioregional',
+}
+
+export class Feature {
+  constructor(
+    private readonly id: string,
+    private readonly tag: FeatureTag,
+    private readonly fragments: FeatureFragment[],
+    private readonly name: string,
+  ) {}
+
+  split(byKey: string): CustomFeature[] {
+    if (this.tag !== FeatureTag.Bioregional) {
+      throw new Error(`Only Bioregional features can be split.`);
+    }
+
+    return this.fragments
+      .filter((fragment) => fragment.key === byKey)
+      .map(
+        (fragment) =>
+          new CustomFeature(
+            `${this.name} / ${fragment.value}`,
```

Comment:
I'd argue that this is correct and preferable in general, but the designs call for `<species> in <split of bioregional>` assuming that the users will do something sensible with these splits - I'd stick to that for the moment (except we wanted to use `/` instead of `in` to have fewer possible translation snags). In practice this means only using `${fragment.value}` here.

Reply:
minor, I am unsure about introducing new lexicon here - `fragment` doesn't feel quite right.

these are all basically subsets based on specific conditions.